server/boot/user.js

```diff
@@ -11,24 +9,18 @@ import {
   frontEndLibsId,
   jsAlgoDataStructId,
   dataVisId,
+  dataViz2018Id,
```

Comment:
`dataViz2018Id` may be cleaner calling it as `dataVizModernId`

Prefer, doing this, here and everywhere?

Reply:
My thinking was, we can overhaul the curriculum again in three years time, what would be call it then?

So I though a yearly name-space would help any further expansion in years to come

Reply:
Hmm.. seems reasonable. Thanks, I'll pull down an check soon.